### ISSUE TITLE
ESBMC 6.3 hangs on minimal C++ example

### ISSUE THREAD

**Issue by vlstill** [esbmc/esbmc#219]
Using the pre-build binary of ESBMC 6.3, I have tried to use it for analysis of C++. However, EBSMC hangs just after the "file ?? Parsing" message is printed:

```
$ ./esbmc test.cpp 
ESBMC version 6.3.0 64-bit x86_64 linux
file test.cpp: Parsing
[nothing happens]
```

From `strace` it appearch ESBMC is accidently waiting for STDIN. Indeed, when I pipe `/dev/null` to ESBMC, it ends (but does not work):

```
./esbmc test.cpp  </dev/null
ESBMC version 6.3.0 64-bit x86_64 linux
file test.cpp: Parsing
Converting
main symbol `main' not found
CONVERSION ERROR
```

`test.cpp`:

```cpp
int main() {
}
```

The same program renamed to `test.c` works correctly. The same problem is also in version 6.2.

**Comment by mikhailramalho** [comment]
Thanks for the report @vlstill, we are aware of it but it's being a tough one to fix. @rafaelsamenezes is currently working on it.

**Comment by mikhailramalho** [comment]
Fixed by c557236cc.

It will be available in the next version of ESBMC.

**Comment by vlstill** [comment]
Thank you very much for a quick solution to this problem.
